extendr-api/src/wrapper/dataframe.rs

```diff
@@ -7,21 +7,32 @@ pub trait IntoDataFrameRow<T> {
 #[derive(PartialEq, Clone)]
 pub struct Dataframe<T> {
     pub(crate) robj: Robj,
-    marker: std::marker::PhantomData<T>,
+    _marker: std::marker::PhantomData<T>,
+}
+
+impl<T> From<Dataframe<T>> for Robj {
+    fn from(value: Dataframe<T>) -> Self {
+        value.robj
+    }
+}
+
+impl<T> FromRobj<'_> for Dataframe<T> {
+    fn from_robj(robj: &Robj) -> std::result::Result<Self, &'static str> {
+        robj.try_into().map_err(|_| "expected a `data.frame`")
```

Comment:
Can you use `.or()` here since the error values is a (constant) static string?

Reply:
Sure! What's the difference?